opus/application/static_media/js/opus.js

```diff
@@ -41,18 +41,20 @@ var opus = {
 
     // client side prefs, changes to these *do not trigger results to refresh*
     // prefs gets added verbatim to the url, so don't add anything weird into here!
-    prefs:{ 'view':'search', // search, browse, cart, detail
-            'browse':'gallery', //either 'gallery' or 'data'
-            'cart_browse':'gallery',  // which view is showing on the cart page, gallery or data
-            'page':default_pages,  // what page are we on, per view, default defined in header.html
-                                   // like {"gallery":1, "data":1, "cart_gallery":1, "cart_data":1 };
-            'limit': 100, // results per page
-            'order': default_sort_order.split(','),  // result table ordering
-            'cols': default_columns.split(','),  // default result table columns by slug
-            'widgets':[], // search tab widget columns
-            'detail':'', // opus_id of detail page content
-
-
+    // prefs key:value pair order has been re-organized to match up with normalized url
+    prefs: {
+        "cols": default_columns.split(","),  // default result table columns by slug
+        "widgets": [], // search tab widget columns
+        "order": default_sort_order.split(","),  // result table ordering
+        "view": "search", // search, browse, cart, detail
+        "browse": "gallery", // either 'gallery' or 'data'
+        "cart_browse": "gallery",  // which view is showing on the cart page, gallery or data
+        "startobs": 1, // for this branch it will not get updated
+        "cart_startobs": 1, // for this branch it will not get updated
+        "detail": "", // opus_id of detail page content
+        "page": default_pages,  // what page are we on, per view, default defined in header.html
```

Comment:
can you add a comment that this will go away w/the startobs branch integration?

Reply:
I don't understand this comment. What will go away? startobs? page? cart_startobs?